atomicapp/cli/main.py

```diff
@@ -37,10 +37,12 @@
 
 logger = logging.getLogger(__name__)
 
-APP_MESSAGE = """
-Your application resides in %s
-Please use this directory for managing your application
-"""
+
+def print_user_msg(app_path):
+    if app_path.startswith('/host'):
+        app_path = app_path[5:]
+    print("\nYour application resides in %s" % app_path)
+    print("Please use this directory for managing your application\n")
```

Comment:
@rtnpro see the comments here https://github.com/projectatomic/atomicapp/pull/345#discussion_r42515514